### ISSUE TITLE
[Keras 3 OpenVINO backend] Implement cross-entropy loss operations for NumPy Parity

### ISSUE THREAD

**Issue by CityBoy-Claude** [openvinotoolkit/openvino#34248]
### Request Description

This issue tracks the implementation of the categorical_crossentropy(), sparse_categorical_crossentropy(), and binary_crossentropy() operations in the Keras OpenVINO backend. This will help increase the coverage of NumPy parity for the OpenVINO backend.

I will submit a PR implementing this and enabling the corresponding tests.

Ref: https://github.com/openvinotoolkit/openvino/issues/34017

### Issue submission checklist

- [x] The feature request or improvement must be related to OpenVINO
